sub/sd_ass.c

```diff
@@ -434,6 +434,12 @@ static void configure_ass(struct sd *sd, struct mp_osd_res *dim,
 #if LIBASS_VERSION >= 0x01306000
     if ((converted || opts->ass_style_override) && opts->ass_justify)
         set_force_flags |= ASS_OVERRIDE_BIT_JUSTIFY;
+#endif
+#if LIBASS_VERSION >= 0x01600010
+    if (converted && !opts->sub_vsfilter_bidi_compat) {
+        track->styles->Encoding = -1;
+        ass_track_set_feature(track, ASS_FEATURE_INCOMPATIBLE_EXTENSIONS, 1);
```

Comment:
This only changes `styles[0]` which is the built-in `Default` style only taking effect if there are no styles at all. FFmpeg converted subs usually define their own `Default` style, and this would be a noop. Instead it should loop over all styles up to `track->n_styles - 1`.

Reply:
Changed. Thanks